index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
- [ OPTIONAL] Please use present tense to write commits like add html file add style.css
![image](https://user-images.githubusercontent.com/74744787/159243236-6983e595-ae20-48c6-aaee-6fc203e3ccf8.png)

Reply:
Will do that...thanks